src/less/progress-bar/base/progress-bar.less

```diff
@@ -0,0 +1,25 @@
+.progress {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: none;
+    //for IE support
+    color: #3665f3;
+    height: 4px;
+    width: 363px;
+}
+
+.progress::-webkit-progress-bar {
+    background-color: #e5e5e5;
```

Comment:
We should probably specify background-color and radius in the class (without prefixes) too...?